lib/git.js

```diff
@@ -13,9 +13,9 @@ const FETCH_DEPTH = 10;
 
 const COMMON_ARGS = [
   "-c",
-  "user.name=GitHub",
+  "user.name=auto-merge[bot]",
   "-c",
-  "user.email=noreply@github.com"
+  "user.email=3579+auto-merge[bot]@users.noreply.github.mlbam.net"
```

Comment:
These are obviously temporary changes, while I figure out how best to support user-provided values. Not strictly necessary for this PR, but would be *very* nice to have.